windows/test-app.js

```diff
@@ -94,6 +94,122 @@ function findUserProjects(projectDir, projects = []) {
   }, projects);
 }
 
+/**
+ * Finds NuGet dependencies.
+ *
+ * Visual Studio (?) currently does not download transitive dependencies. This
+ * is a workaround until `react-native-windows` autolinking adds support.
```

Comment:
do we have an estimate on when this might happen? Is it on the RNWindows team radar?

Reply:
I believe this is the bug: https://github.com/microsoft/react-native-windows/issues/9578